addons/base_import/static/tests/import_action_tests.js

```diff
@@ -546,6 +546,42 @@ QUnit.module("Base Import Tests", (hooks) => {
         );
     });
 
+    QUnit.test("Import view: drag-and-drop file support", async function (assert) {
+        registerFakeHTTPService((route, params) => {
+            assert.strictEqual(route, "/base_import/set_file");
+            assert.strictEqual(
+                params.ufile[0].name,
+                "fake_file.csv",
+                "file is correctly uploaded to the server"
+            );
+        });
+        await createImportAction();
+
+        // Simulate a file drag and drop:
+        const dataTransfer = new DataTransfer();
+        dataTransfer.items.add(new File(["fake_file"], "fake_file.csv", {
+            type: "text/plain"
+        }));
+        const container = target.querySelector(".o_import_action");
+        container.dispatchEvent(new DragEvent("dragenter", {
+            bubbles: true,
+            dataTransfer
+        }));
+        await nextTick();
+        const dropzone = target.querySelector(".o-mail-Dropzone");
```

Comment:
"o-mail-Dropzone"? Seems like a mail-specific class, not sure what it has to do with anything here.

Reply:
It's because the `useDropzone` is defined in the `mail` module and the discuss team uses a nomenclature "<module_name>-<component_name>" for their class names. Maybe this has to be moved to `web`?

**Actually, I should maybe do the move now because `base_import` depends on `web` only and `web` depends on `base` only and `base` has no dependency. So, it's possible that the import will be undefined as we may not have `mail` installed.**